server/database.sql

```diff
@@ -1,6 +1,6 @@
-CREATE DATABASE pernprueba;
+-- CREATE DATABASE pernprueba;
 
-\c pernprueba;
+-- \c pernprueba;
```

Comment:
If you no longer need this, can you remove it?

Reply:
Yes, deleted aswell